trace/opencensus/trace/labels_helper.py

```diff
@@ -0,0 +1,101 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from opencensus.trace.reporters import google_cloud_reporter
+
+import os
+
+# Environment variable set in App Engine when vm:true is set.
+_APPENGINE_FLEXIBLE_ENV_VM = 'GAE_APPENGINE_HOSTNAME'
+
+# Environment variable set in App Engine when env:flex is set.
+_APPENGINE_FLEXIBLE_ENV_FLEX = 'GAE_INSTANCE'
+
+_GAE_PROJECT_ENV = 'GCLOUD_PROJECT'
```

Comment:
Maybe add a link to https://cloud.google.com/appengine/docs/flexible/python/runtime#environment_variables